packages/manager/modules/pci/src/projects/creating/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,16 @@
+{
+  "pci_projects_creating_main_title": "Vous êtes en route pour l’aventure Public Cloud",
+  "pci_projects_creating_secondary_title": "La création de votre projet est en cours",
+  "pci_projects_creating_explain1": "Pour finaliser la création de votre projet, veuillez effectuer le paiement du bon de commande <a href=\"{{ href }}\">{{ orderId }}</a>.",
+  "pci_projects_creating_explain2": "Si cette page est toujours présente plusieurs minutes après le paiement du bon de commande, merci de contacter <a href=\"{{ href }}\">notre service client</a>.",
+  "pci_projects_creating_finalize": "Finaliser ma commande",
+  "pci_projects_creating_finalize_title": "{{ orderId }} : finaliser ma commande",
```

Comment:
Looks weird as a feedback message

Reply:
This is copy paste ^^ In fact some keys are not used. I remove them